packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -367,16 +367,18 @@ const Services = WebexPlugin.extend({
 
   /**
    * simplified method to update the preauth catalog via email
-   * @param {object} param
-   * @param {string} param.email - must be a standard-format email
+   *
+   * @param {object} query
+   * @param {string} query.email - A standard format email.
+   * @param {string} query.orgId - The user's OrgId.
    * @returns {Promise<void>}
    */
-  collectPreauthCatalog({email} = {}) {
-    if (!email) {
+  collectPreauthCatalog(query) {
```

Comment:
This will be a breaking change for the web client(s). Have we may any further progress of not releasing everything as a minor/patch?

Reply:
@andrewholsted, i think it still supports email. it looks like this change is to support orgId too

Reply:
@aimeex3 is correct. Let me know if this presents a problem during any testing.